integration-test/src/main/resources/application.yml

```diff
@@ -138,6 +138,9 @@ integrationtest:
         assumerIdentity:
         loggerIdentity:
       secure: false
+    resourcegroup:
+      usage: MULTIPLE
+      name:
```

Comment:
is name not mandatory?
will single be tested as well?

Reply:
I added the default values as of now. We need to test the single RG scenario. Once its done, we will set the test to: SINGLE. 
The name parameter: it is not mandatory. With usage MULTIPLE it is not allowed to provide it. With single it is optional. If provided, cloudbreak will use that RG, if not, then cloudbreak will create one and use it (latter use case is a demo use case, the real advantage comes when the customer provides an existing RG)